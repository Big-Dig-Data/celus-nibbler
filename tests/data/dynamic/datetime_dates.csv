Date,Sessions
2020-01-01 10:20:30,1
2020-01-05 11:22:33,2
2020-01-09 11:22:33,4
