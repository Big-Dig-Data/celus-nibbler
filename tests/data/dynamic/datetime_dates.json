
{
  "kind": "non_counter.generic",
  "version": 1,
  "parser_name": "datetime_dates",
  "data_format": {
    "name": "simple_format"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [],
  "dimension_aliases": [],
  "heuristics": {
    "conds": [
      {
        "pattern": "Date",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "data_headers": {
        "roles": [
            {
                "source": {
                    "coord": {
                        "row": 0,
                        "col": 1
                    },
                    "direction": "right"
                },
                "role": "metric"
            }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 1,
            "col": 1
          },
          "direction": "right"
        }
      },
      "dates": {
        "source": {
          "coord": {
            "row": 1,
            "col": 0
          },
          "direction": "down"
        },
        "date_pattern": "%Y-%m-%d %H:%M:%S"
      },
      "title_ids": [],
      "dimensions": [],
      "aggregate_same_records": true,
      "kind": "non_counter.generic"
    }
  ]
}
