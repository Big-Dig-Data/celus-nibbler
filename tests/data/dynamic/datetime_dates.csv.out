"2020-01-01","2020-01-31","","","","","","","","","Sessions","7"
